readme/MacBuild.md

```diff
@@ -38,33 +179,10 @@ Follow these steps to resolve the issue:
 3. Exit the shell with `exit`
 4. Continue the build with option 2
 
-### 2. Start a jhbuild shell
-````bash
-$HOME/.new_local/bin/jhbuild shell
-````
-
-### 3. Build Poppler
-Execute in this folder.
-````bash
-./build-poppler.sh
-````
-
-### 4. Build PortAudio
-
-````bash
-./build-portaudio.sh
-````
-
-### 5. Build LibZip
-
-````bash
-./build-libzip.sh
-````
-#### Potential Errors
-##### Unknown module (macOS Mojave 10.14)
+#### Unknown module error (macOS Mojave 10.14)
 
 If there is an error like:
-````bash
+````sh
```

Comment:
Should we really still include comments about macOS Mojave?

Reply:
This is copied over from the old readme. Not sure if this issue still occurs...

Reply:
I deleted this since it is likely that this issue has been resolved in jhbuild a long time ago.